Sources/Models/MessageStyle.swift

```diff
@@ -89,7 +89,7 @@ public enum MessageStyle {
             image = UIImage(cgImage: cgImage, scale: image.scale, orientation: corner.imageOrientation)
         }
 
-        return stretch(image).withRenderingMode(.alwaysTemplate)
```

Comment:
I think `.withRenderingMode(.alwaysTemplate)` is necessary, no? We want to ignore the color of the images so that we can apply a different bubble color or border color. The image is black.

Reply:
@SD10 , Nope, we just use the image to mask the `containerView`, so it's just use maskView's alpha channel.

Reply:
@zhongwuzw I don't know what I would do without you 😅

Reply:
By the way, we should optimize that and not use `mask`, it's performance penalty.

Reply:
Yes, I agree. I'm trying to remove the images altogether. See #307 #324. I can't figure out a good API. I will work on this more